tests-e2e/api/job-postings.spec.js

```diff
@@ -132,38 +177,62 @@ describe('JobPostings API Endpoints', () => {
                 });
         });
 
-        it('devrait pouvoir modifier la pagination avevc le paramètre de requête "pagination"', async () => {
+        it('devrait pouvoir modifier la pagination avec les paramètres de requête "pagination"', async () => {
             expect.hasAssertions();
             await frisby
                 .get(
-                    `http://api:3001/api/job-postings?pagination=${JSON.stringify(
-                        [2, 1]
-                    )}`
+                    `http://api:3001/api/job-postings?${querystring.stringify({
+                        perPage: 2,
+                        currentPage: 1,
+                    })}`
                 )
                 .expect('status', 200)
                 .expect(
                     'header',
                     'content-type',
                     'application/json; charset=utf-8'
                 )
-                .expect('header', 'content-range', 'jobpostings 0-2/3')
-                .then(resp => {
+                .expect('header', 'x-total-count', '3')
+                .expect(
+                    'header',
+                    'link',
+                    [
+                        '</api/job-postings?currentPage=1&perPage=2>; rel="first"',
+                        // '</api/job-postings?currentPage=1&perPage=2>; rel="prev"',
+                        // '</api/job-postings?currentPage=1&perPage=2>; rel="self"',
+                        // '</api/job-postings?currentPage=2&perPage=2>; rel="next"',
+                        // '</api/job-postings?currentPage=2&perPage=2>; rel="last"'
+                    ].join(',')
+                )
+                .then((resp) => {
                     expect(resp.json.length).toStrictEqual(2);
                 });
             await frisby
                 .get(
-                    `http://api:3001/api/job-postings?pagination=${JSON.stringify(
-                        [2, 2]
-                    )}`
+                    `http://api:3001/api/job-postings?${querystring.stringify({
+                        perPage: 2,
+                        currentPage: 2,
+                    })}`
                 )
                 .expect('status', 200)
                 .expect(
                     'header',
                     'content-type',
                     'application/json; charset=utf-8'
                 )
-                .expect('header', 'content-range', 'jobpostings 2-3/3')
-                .then(resp => {
+                .expect('header', 'x-total-count', '3')
+                .expect(
+                    'header',
+                    'link',
+                    [
+                        '</api/job-postings?currentPage=1&perPage=2>; rel="first"',
+                        // '</api/job-postings?currentPage=1&perPage=2>; rel="prev"',
```

Comment:
If it's not being used, it's probably best to erase it?

Reply:
Alors en fait je peux effectivement tester chaque élément mais seulement de cette manière :
```javascript
.expect(
    'header',
    'link',
    '</api/job-postings?currentPage=1&perPage=10>; rel="first"'
)
.expect(
    'header',
    'link',
    '</api/job-postings?currentPage=1&perPage=10>; rel="prev"'
)
.expect(
    'header',
    'link',
    '</api/job-postings?currentPage=1&perPage=10>; rel="self"'
)
.expect(
    'header',
    'link',
    '</api/job-postings?currentPage=1&perPage=10>; rel="next"'
)
.expect(
    'header',
    'link',
    '</api/job-postings?currentPage=1&perPage=10>; rel="last"'
)
```